q2_quality_filter/test/test_filter.py

```diff
@@ -320,6 +320,16 @@ def test_stats_to_metadata(self):
         self.assertEqual(obs.column_count, 5)
         self.assertEqual(obs.id_header, 'sample-id')
 
+    def test_numeric_ids(self):
+        filepath = self.get_data_path('stats-numeric.txt')
+        format = QualityFilterStatsFmt(filepath, mode='r')
+        transformer = self.get_transformer(QualityFilterStatsFmt,
+                                           qiime2.Metadata)
+        obs = transformer(format)
+        self.assertEqual(obs.id_count, 34)
+        self.assertEqual(obs.column_count, 5)
+        self.assertEqual(obs.id_header, 'sample-id')
```

Comment:
Should there be a test here that the values in the `sample-id` column are strings?

Reply:
Scratch that, I see now that `Metadata` would fail if the ids were not strings.